### PR TITLE
Fixed showing options bug

### PR DESCRIPTION
**Pull request by RemcovdMonta**

### CODE REVIEW

File: view/frontend/web/js/view/checkout/shipping/additional-block.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ define(
                                     this.preferredShipper = objectArray[0];
                                 }
 
-                                const filteredDeliveryServicesList = objectArray.filter(timeframe => timeframe.options[0].date !== '');
```
Comment (WilcoMonta) [inline view/frontend/web/js/view/checkout/shipping/additional-block.js:234]: whitespace nog even weghalen

File: view/frontend/web/js/view/checkout/shipping/additional-block.js [hunk 2]
```diff
@@ -280,35 +279,30 @@ define(
...
                             "day": option.options[0].date_string.split(' ')[0],
-                            "day_string":
-                                option.options[0].date_string.split(' ')[1].concat(' ', option.options[0].date_string.split(' ')[2]),
+                            "day_string": option.options[0].date_string.split(' ')[1].concat(' ', option.options[0].date_string.split(' ')[2]),
                             "discount_percentage" : option.options.some(x=>x.discount_percentage > 0) ? option.options.find(x=>x.discount_percentage > 0).discount_percentage : 0,
                             "discount_percentage_text" : option.options.some(x=>x.discount_percentage > 0) ? '-' + option.options.find(x=>x.discount_percentage > 0).discount_percentage + '%' : 0
                         }
                     });
-                    
+
```
Comment (WilcoMonta) [inline view/frontend/web/js/view/checkout/shipping/additional-block.js:299]: whitespace weghalen

**Comment by leonhelmus** [comment]
@RemcovdMonta does this patch also resolve the following error i see in the logs:
[2023-06-16T09:05:53.381148+00:00] report.CRITICAL: Exception: Warning: Trying to access array offset on value of type null in /vendor/monta/montacheckout/Plugin/Quote/Model/Quote/Address/Total/Shipping.php on line 99 in /vendor/magento/framework/App/ErrorHandler.php:61

**Comment by RemcovdMonta** [comment]
@leonhelmus This error has been fixed in #61, I had to make the fix more general as otherwise an earlier issue reappeared.

**Comment by leonhelmus** [comment]
@RemcovdMonta thank you very much! I will deploy the fix today and will let you know the results.

**Pull request merged**